app/src/main/java/com/kks/nimblesurveyjetpackcompose/repo/login/LoginRepoImpl.kt

```diff
@@ -45,6 +45,7 @@ class LoginRepoImpl @Inject constructor(
                         PREF_REFRESH_TOKEN,
                         loginResponse.attributes?.refreshToken
                     )
+                    preferenceManager.setBooleanData(PREF_LOGGED_IN, true)
```

Comment:
Just a thought, do we really need this preference? Why not simply check if `PREF_REFRESH_TOKEN` exists so that we don't have to update this if ever? 🤔

Reply:
@ryan-conway I've removed and made use of `PREF_REFRESH_TOKEN` like you suggested in `Integrate PR`